charts/model-engine/templates/service_config_map.yaml

```diff
@@ -11,46 +12,110 @@ metadata:
+    # NOTE: If you add/change values inside this file that need to apply to all clusters, please make changes in
+    # all service_config_{env}.yaml files as well.
 
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: {{ include "modelEngine.fullname" . }}-service-config
-  namespace: {{ .Values.config.values.launch.endpoint_namespace }}
-  labels:
-    {{- include "modelEngine.labels" . | nindent 4 }}
-  annotations:
-    "helm.sh/hook": pre-install,pre-upgrade
-    "helm.sh/hook-weight": "-2"
-data:
-  launch_service_config: |-
-    dd_trace_enabled: {{ .Values.dd_trace_enabled | default false | quote }}
-    gateway_namespace: {{ .Release.Namespace | quote }}
-    {{- with .Values.config.values.launch }}
-    {{- range $key, $value := . }}
-    {{ $key }}: {{ $value | quote }}
-    {{- end }}
-    {{- end }}
+    # Config for scale-hosted Hosted Model Inference in the prod cluster, see `service_config` for more details
+    model_primitive_host: model-server.{{ .Release.Namespace }}.svc.cluster.local
+
+    # # Endpoint config
+    # K8s namespace the endpoints will be created in
+    endpoint_namespace: {{ .Release.Namespace | quote }}
+
+    # Asynchronous endpoints
+    sqs_profile: {{ $.Values.serviceAccount.sqsProfileName }}
+    sqs_queue_policy_template: |-
+      {
+          "Version": "2012-10-17",
+          "Id": "__default_policy_ID",
+          "Statement": [
+          {
+              "Sid": "__owner_statement",
+              "Effect": "Allow",
+              "Principal": {
+              "AWS": "arn:{{ .Values.aws.partition }}:iam::{{ .Values.aws.accountId }}:root"
+              },
+              "Action": "sqs:*",
+              "Resource": "arn:{{ .Values.aws.partition }}:sqs:{{ .Values.aws.region }}:{{ .Values.aws.accountId }}:${queue_name}"
+          },
+          {
+              "Effect": "Allow",
+              "Principal": {
+              "AWS": "arn:{{ .Values.aws.partition }}:iam::{{ .Values.aws.accountId }}:role/{{ $.Values.serviceAccount.sqsProfileName }}"
+              },
+              "Action": "sqs:*",
+              "Resource": "arn:{{ .Values.aws.partition }}:sqs:{{ .Values.aws.region }}:{{ .Values.aws.accountId }}:${queue_name}"
+          },
+          {
+              "Effect": "Allow",
+              "Principal": {
+              "AWS": "arn:{{ .Values.aws.partition }}:iam::{{ .Values.aws.accountId }}:role/ml_hosted_model_inference"
+              },
+              "Action": "sqs:*",
+              "Resource": "arn:{{ .Values.aws.partition }}:sqs:{{ .Values.aws.region }}:{{ .Values.aws.accountId }}:${queue_name}"
+          }
+          ]
+      }
+
+    sqs_queue_tag_template: |-
+      {
+          "infra.scale.com/product": "{{ .Values.productTag }}",
+          "infra.scale.com/team": "${team}",
+          "infra.scale.com/contact": "{{ .Values.contactEmail }}",
+          "infra.scale.com/customer": "AllCustomers",
+          "infra.scale.com/financialOwner": "{{ .Values.contactEmail}}",
```

Comment:
need to template "infra.scale.com" to something or allow these as additional tags set in the values.yaml